keyboards/keebio/iris/keymaps/via/keymap.c

```diff
@@ -65,3 +65,19 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
                                 // └────────┴────────┴────────┘                 └────────┴────────┴────────┘
   )
 };
+
+#ifdef ENCODER_MAP_ENABLE
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+#   if defined(KEYBOARD_keebio_iris_rev7)
```

Comment:
Looking for some feedback here on if this is the right approach for handling differing numbers of encoders

Reply:
Not sure there is a great way to handle this, aside from having the keymap folder in the different revision folders. 

But this should be okay, I think.